sdk/lib/atl/atldef.h

```diff
@@ -7,6 +7,14 @@
 
 #pragma once
 
+#ifndef __REACTOS__
+    #ifdef __cplusplus
+        #include <cstddef>
```

Comment:
Is this necessary? C++ supports <stddef.h> just fine

Reply:
Moved to `CAtlFileMapping.cpp`.